### PR TITLE
README: Describe how to use the bot

### PR DESCRIPTION
**Pull request by orgads**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,18 @@ The servers are written in Typescript.
   A simple server that accepts websocket connections and sends pre-set transcriptions to the client.
 * [Bot API](ac-bot-api) \[[API documentation](https://techdocs.audiocodes.com/voice-ai-connect/#VAIG_API/API_1.htm)\]:
   A server that echos incoming messages. It also supports pre-defined replies like "disconnect", which hangs up.
+  To use the bot, you need to set the environment variable ACCESS_TOKEN for running the bot, and configure it accordingly in VoiceAI Connect.
+  For example, if the token is MyS3cr3tT0k3n, the configuration should look like this:
+  ```json
+  {
```
Comment (yoshigev) [inline README.md:22]: Put it inside `providers` field with an array.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,19 @@ The servers are written in Typescript.
 * [Bot API](ac-bot-api) \[[API documentation](https://techdocs.audiocodes.com/voice-ai-connect/#VAIG_API/API_1.htm)\]:
   A server that echos incoming messages. It also supports pre-defined replies like "disconnect", which hangs up.
 
+  To use the bot, you need to set the environment variable ACCESS_TOKEN for running the bot, and configure it accordingly in VoiceAI Connect.
```
Comment (yoshigev) [inline README.md:18]: Add blank line before.

File: README.md [hunk 3]
```diff
@@ -15,6 +15,19 @@ The servers are written in Typescript.
...
+  For example, if the token is MyS3cr3tT0k3n, the configuration should look like this:
```
Comment (yoshigev) [inline README.md]: ditto

Comment (yoshigev) [inline README.md]: s/configuration/VoiceAI Connect configuration/

File: README.md (hunk 1 above)
Comment (orgads) [reply README.md:22]: Done

File: README.md (hunk 2 above)
Comment (orgads) [reply README.md:18]: Done

File: README.md (hunk 3 above)
Comment (orgads) [reply README.md]: Done

Comment (orgads) [reply README.md]: Done

**Review by yoshigev — APPROVED** [review]

**Pull request merged**